Source/Common/Config.cpp

```diff
@@ -489,14 +500,18 @@ fextl::string GetDataDirectory() {
   return DataDir;
 }
 
-fextl::string GetConfigDirectory(bool Global) {
+fextl::string GetConfigDirectory(bool Global, const PortableInformation* PortableInfo) {
+  const char* ConfigOverride = getenv("FEX_APP_CONFIG_LOCATION");
```

Comment:
We should avoid querying this in non-portable mode.

Reply:
With portable mode we are still allowing `FEX_APP_CONFIG_LOCATION` to override the path if it is set.

Reply:
Ah, I missed that the variables already existed before. Looks fine then.